assets/src/components/selection/event-select/index.js

```diff
@@ -42,27 +43,35 @@ export const EventSelect = ( {
 	selectedEventId,
 	isLoading,
 } ) => {
-	if ( isLoading || isEmpty( events ) ) {
-		return <Placeholder key="placeholder"
-			icon="calendar"
-			label={ __( 'EventSelect', 'event_espresso' ) }
-		>
-			{ isLoading ?
-				<Spinner /> :
-				__(
-					'There are no events to select from. You need to create an event first.',
-					'event_espresso',
-				)
-			}
-		</Placeholder>;
+	if ( isEmpty( events ) ) {
+		return (
+			<Fragment>
+				<Placeholder
+					icon="calendar"
+					label={ __( 'EventSelect', 'event_espresso' ) }
+				>
+					{ ! isLoading && isEmpty( events ) ?
+						__(
+							'There are no events to select from. You need to create an event first.',
+							'event_espresso',
+						) :
+						<Spinner />
```

Comment:
I'm assuming `<Spinner />` is a self closing JSX tag and does not require any other attributes?
(and yes I know that it was imported from `@wordpress/components`)

Reply:
correct.